functions/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "functions",
   "description": "Cloud Functions for Firebase",
+  "engines": { "node": "8" },
```

Comment:
Y ya que estamos en esto... pq no pasar a 10 al toque?

Reply:
Muy válida duda, por comodidad, la ignoraré :sweat_smile: Pero for the record, creo que estás en lo correcto que 10 de una hubiera sido mejor.